mmv1/products/developerconnect/InsightsConfig.yaml

```diff
@@ -0,0 +1,248 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: InsightsConfig
+description: Description
+base_url: projects/{{project}}/locations/{{location}}/insightsConfigs
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/insightsConfigs/{{insights_config_id}}
+create_url: projects/{{project}}/locations/{{location}}/insightsConfigs?insightsConfigId={{insights_config_id}}
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/insightsConfigs/{{insights_config_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/insightsConfigs/{{insights_config_id}}
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'developer_connect_insights_config_basic'
```

Comment:
This test won't currently pass, as the config is a stub. I'd recommend we create a complete example including dependencies like the app hub application, similar to the handwritten tests. This is standard for Terraform docs.